snuba/query/mql/parser_supported_join.py

```diff
@@ -1315,6 +1314,10 @@ def populate_query_from_mql_context(
             else:
                 query.set_ast_groupby([selected_time.expression])
 
+        if query.get_groupby():
+            # Only set WITH TOTALS if there is a group by.
```

Comment:
Is this same issue applicable to `HAVING` clause? The [documentation](https://clickhouse.com/docs/en/sql-reference/statements/select/group-by#with-totals-modifier) says that the behavior in case of `HAVING` depends on `totals_mode` setting and we have a few occurrences of `HAVING` clause in snuba code base.

Reply:
The `HAVING` clause would not apply here since MQL doesn't support it

Reply:
Thanks for the clarification!